js_modules/dagit/packages/core/src/ui/Checkbox.tsx

```diff
@@ -12,7 +12,9 @@ type Props = React.DetailedHTMLProps<
   HTMLInputElement
 > & {
   checked: boolean;
-  label: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  labelText?: string;
+  label?: React.ReactNode;
```

Comment:
This seems a little odd because label: ReactNode can also be a string? Seems one should be sufficient?